internal/registration/registration.go

```diff
@@ -131,3 +217,12 @@ func (r *Registration) Deregister() error {
 	r.registered = false
 	return errors
 }
+
+type YGGDresponse struct {
```

Comment:
why not to have it next to other types at the top of the file?

Reply:
nit: s/YGGDresponse/YGGDResponse